app/utils/keyBindings.js

```diff
@@ -27,12 +20,10 @@ export const handleRememberArticle = (store) => {
     const article = getArticle(state, url);
     const user = getUser(state);
 
-    updateDuplicationConfirmationState(url);
-
     if (!user) {
       showNotification('Please, open "Options" window and log in.');
-    } else if (article.get('state') === true) {
-      showNotification('Article is already saved. Use extension\'s popup for duplication.');
+    } else if (article.get('id')) {
```

Comment:
It's removed already :see_no_evil: : https://github.com/heutagogy/heutagogy-chrome-extension/commit/ee9468aaf611b5027de56f79776e7fa10ffad306#diff-ccdf8d36c3c6512689ce5b02bfdcb3ffL5